src/lib.rs

```diff
@@ -59,6 +93,8 @@ impl slog::Drain for TracingSlogDrain {
                     (&keys.file, Some(&record.file())),
                     (&keys.line, Some(&record.line())),
                     (&keys.column, Some(&record.column())),
+                    #[cfg(feature = "kv")]
+                    (&keys.fields, Some(&field_serializer.as_str())),
```

Comment:
I wonder if this could/should be `None` if .as_str() returns empty. Maybe even return Option<> from .as_str() instead.

Reply:
Good point! I added additional logic to the `as_str` function and a test to verify that we don't emit this field if there are not any key-value pairs in the original log.